tools/codegen/pkg/bootstrapgen/template/bootstrapTemplate.go

```diff
@@ -87,7 +87,7 @@ var (
 								}
 							}
 						{{else}}
-							if cpb.{{.GoName}} == "" {
+							if cpb.{{.GoName}} == "" || cpb.{{.GoName}} == "\"\"" {
```

Comment:
Can you update this to use a constant for the empty string:
```
const emptyStr = `""`
```

or similar

Reply:
Ideally it'd clean up the golden file too. The "\"\"" everywhere just looks ugly, hard to read.

Reply:
D